prow/cmd/clonerefs/BUILD.bazel

```diff
@@ -15,10 +15,21 @@ go_library(
 )
 
 container_image(
-    name = "image",
+    name = "base-image",
     base = "@git-base//image",
     directory = "/",
-    files = [":clonerefs"],
+    visibility = ["//visibility:public"],
+)
+
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "image",
+    base = ":base-image",
+    embed = [":go_default_library"],
+    goarch = "amd64",
+    goos = "linux",
+    pure = "on",
```

Comment:
Is it important that the `clonerefs` image is located at `/`? If so, I will revert changes to these pod utilities as I think this may be a bit tricky and out-of-scope.

Reply:
it is important, the pod-utils hard-code this location